### ISSUE TITLE
yarn create umi 的项目 依赖有好多错误，不怕有BUG吗

### ISSUE THREAD

**Issue by 15207799874** [umijs/umi#1531]
yarn create umi
yarn
npm ls

npm ERR! peer dep missing: eslint@^2.10.2 || ^3 || ^4, required by eslint-plugin-jsx-a11y@5.1.1
npm ERR! peer dep missing: babel-eslint@^7.2.3, required by eslint-config-react-app@2.1.0
npm ERR! peer dep missing: eslint@^4.1.1, required by eslint-config-react-app@2.1.0
npm ERR! extraneous: babel-core@7.0.0-bridge.0 D:\vue react\umi-test\node_modules\babel-jest\node_modules\babel-core
npm ERR! peer dep missing: dva@^2.2.0-0, required by dva-immer@0.2.4
npm ERR! peer dep missing: redbox-react@1.x, required by babel-plugin-dva-hmr@0.4.1
npm ERR! peer dep missing: dva-core@^1.1.0, required by dva-loading@2.1.0-beta.2

**Comment by sorrycc** [comment]
peer dep missing 忽略吧，和用法有关。
